server/app/views/applicant/NorthStarApplicantUpsellView.java

```diff
@@ -20,92 +20,97 @@
-
-    ProgramSectionParams cardsSection =
-        programCardsSectionParamsFactory.getSection(
-            params.request(),
-            params.messages(),
-            Optional.empty(),
-            MessageKey.BUTTON_VIEW_AND_APPLY,
-            params.eligiblePrograms().get(),
-            /* preferredLocale= */ params.messages().lang().toLocale(),
-            Optional.of(params.profile()),
-            Optional.of(params.applicantId()),
-            params.applicantPersonalInfo(),
-            ProgramCardsSectionParamsFactory.SectionType.STANDARD);
-    context.setVariable("cardsSection", cardsSection);
-
-    return templateEngine.process("applicant/ApplicantUpsellTemplate", context);
-  }
+    private final ProgramCardsSectionParamsFactory programCardsSectionParamsFactory;
+
+    @Inject
+    NorthStarApplicantUpsellView(
+            TemplateEngine templateEngine,
+            ThymeleafModule.PlayThymeleafContextFactory playThymeleafContextFactory,
+            AssetsFinder assetsFinder,
+            ApplicantRoutes applicantRoutes,
+            SettingsManifest settingsManifest,
+            LanguageUtils languageUtils,
+            DeploymentType deploymentType,
+            ProgramCardsSectionParamsFactory programCardsSectionParamsFactory) {
+        super(
+                templateEngine,
+                playThymeleafContextFactory,
+                assetsFinder,
+                applicantRoutes,
+                settingsManifest,
+                languageUtils,
+                deploymentType);
+        this.programCardsSectionParamsFactory = checkNotNull(programCardsSectionParamsFactory);
+    }
+
+    public String render(UpsellParams params) {
+        ThymeleafModule.PlayThymeleafContext context = createThymeleafContext(
+                params.request(),
+                Optional.of(params.applicantId()),
+                Optional.of(params.profile()),
+                params.applicantPersonalInfo(),
+                params.messages());
+
+        context.setVariable(
+                "pageTitle", params.messages().at(MessageKey.TITLE_APPLICATION_CONFIRMATION.getKeyName()));
+
+        context.setVariable("programTitle", params.programTitle().orElse(""));
+        context.setVariable("programDescription", params.programDescription().orElse(""));
+        context.setVariable("applicationId", params.applicationId());
+        context.setVariable("bannerMessage", params.bannerMessage());
+
+        ImmutableList<String> additionalText = ImmutableList.of("Text1", "Text2");
+
+        String alertTitle = params
+                .messages()
+                .at(MessageKey.ALERT_SUBMITTED.getKeyName(), params.programTitle().orElse(""));
+        AlertSettings successAlertSettings = AlertSettings.builder()
+                .show(true)
+                .title(Optional.of("Alert Title"))
+                .text("Alert Text")
+                .unescapedDescription(false)
+                .alertType(AlertType.INFO)
```

Comment:
suggestion/question (maybe blocking): please double check, but I think this was `AlertType.SUCCESS` previously.